config/common.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @var $params array
+ */
+
+return [
+    \Sentry\Transport\TransportFactoryInterface::class => \Sentry\Transport\DefaultTransportFactory::class,
+    \Sentry\HttpClient\HttpClientFactoryInterface::class => function (Yiisoft\Injector\Injector $injector) {
+        return $injector->make(\Sentry\HttpClient\HttpClientFactory::class, [
```

Comment:
I think we can safely do `new` here instead of using injector.

Reply:
Not really, but I rewrote syntax to array-like